docs/list_of_tutorials.jl

```diff
@@ -60,6 +60,11 @@ if generate_tutorials
             ],
             "DG Methods" =>
                 ["Filters" => "Numerics/DGMethods/showcase_filters.jl"],
+            "Time-Stepping" => [
+                "Introduction" => "Numerics/TimeStepping/ts_intro.jl",
```

Comment:
I think it needs to be added here:
```suggestion
                "Introduction" => "Numerics/TimeStepping/ts_intro.jl",
                "Numerics/TimeStepping/tutorial_risingbubble_config.jl",
```

Reply:
Makes sense, thank you!